frameworks/digital-outcomes-and-specialists/questions/briefs/technicalCompetenceCriteriaParticipants.yml

```diff
@@ -0,0 +1,28 @@
+name: Technical competence criteria
+question: Technical competence criteria
+question_advice: |
+  Technical competence is how well the supplier can do the work you need them to do.
+
+  Evaluate suppliers’ technical competence using criteria for:
+
+  - essential skills and experience
+  - nice-to-have skills and experience
+  - proposal
+
+  **Scoring (optional)**<br />
+  If you want to score individual criteria, you must add points for everything. Read [how to score individual criteria](https://www.gov.uk/guidance/how-to-set-your-evaluation-criteria-when-buying-digital-outcomes-and-specialists-services#giving-points-to-individual-criteria).
```

Comment:
`you must add points for everything` feels a bit vague and unhelpful.  Could we have an example here, like the one for specialists?  Something like this seems useful:
`You can give points to individual criteria, eg ‘have experience recruiting users with low digital literacy – 15 points’. If you choose to add points, you must do it for all individual criteria.`